applications/OptimizationApplication/python_scripts/utilities/opt_line_search.py

```diff
@@ -59,3 +61,71 @@ def GetInfo(self) -> dict:
                 'scaled_step': self.step}
 
         return info
+    
+class BB_Step(object):
+    @classmethod
+    def GetDefaultParameters(cls):
+        return Kratos.Parameters("""{
+            "type"               : "BB_step",
+            "init_step"          : 0,
+            "max_step"           : 0,
+            "gradient_scaling": ": inf_norm"
```

Comment:
```suggestion
            "gradient_scaling" : "inf_norm"
```

this should've made the tests fail